CMakeLists.txt

```diff
@@ -230,7 +233,11 @@ if(CPPDAP_BUILD_EXAMPLES)
         target_link_libraries(${target} cppdap "${CPPDAP_OS_LIBS}")
 
         if(CPPDAP_INSTALL_VSCODE_EXAMPLES)
-            set(extroot "$ENV{HOME}/.vscode/extensions")
+            if(CMAKE_SYSTEM_NAME MATCHES "Windows")
```

Comment:
I think VS Code can be installed either locally or system-wide on Windows, so it's possible that it won't be under the userprofile. Doesn't need to be fixed now, but something to keep in mind.

Reply:
ack. It does check the directory exists, and warns if it missing.